.pre-commit-config.yaml

```diff
@@ -42,3 +42,4 @@ repos:
     rev: cdefcb096e520a6daa9552b1d4636f5f1e1729cd
     hooks:
     - id: hadolint
+      entry: --entrypoint /bin/hadolint hadolint/hadolint:latest --ignore DL3061 --ignore DL3013 --ignore DL3042
```

Comment:
minor preference to keep the ignores in the Docker file and not here?

Reply:
I tried that, but it couldn't get it to work - the pre-commit hook kept failing even with the ignores in (including the ones in the same places that they were in cohort-extractor)

Reply:
Have removed hadolint from the pre-commit hooks altogether and put it in the `just check` command instead (where it seems to respect the ignores in the Dockerfile fine)